DirectEmployers/MyJobs
# [PD-3385] tjd-distribution-location-and-rule-summary

## What's this PR do?
This PR checks for the rules applied and depending on that basis determines what message to show the user in the Distribution Details box concerning where the emails are going to be sent

## Screenshots (if appropriate)
![screen shot 2017-07-28 at 11 37 38 am](https://user-images.githubusercontent.com/22029077/28724773-336f1990-7389-11e7-8c83-b60834e4754c.png)


## Where should the reviewer start?
Start inside of the TJD Application

## How should this be manually tested?
Pull down the repository. Go into TJD and select one of the Organizations. It would be easier if you selected one that was a custom CBO so you can edit the rule, locations, etc if need be. Go into edit or view a contact. On the Manage contact page you will see the Distribution Details on the right hand side. If you mess with the Rules or Locations and add or delete them, depending on the criteria settings, the message should change depending on what is selected.

## Any background context you want to provide?
N/A
